index.html

```diff
@@ -7,7 +7,7 @@
   <meta name="description" content="rlottie viewer">
   <meta name="keywords" content="lottie, rlottie, lottie viewer, rlottie viewer, prettyview">
   <meta name="author" content="Subhransu, Chae Lin Shin, Sun Hwan Park, Chaewon Shin, Seunghyun Shin, Seungjae Yoon">
-  <title>rlottie Viewer</title>
+  <title>PrettyView</title>
```

Comment:
This should be 'rlottie Viewer' not 'PrettyView'

Reply:
We will update it to rlottie Viewer in the next patch.